terratest/src/test/pega/data/values_with_customcerts.yaml

```diff
@@ -0,0 +1,45 @@
+---
+global:
+  certificates:
+    testcert.cer: |
+      ----THIS IS MY CERT----
+  tier:
+    - name: "web"
+      nodeType: "WebUser"
+      requestor:
+        passivationTimeSec: 900
+      replicas: 1
+      deploymentStrategy:
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
+        type: RollingUpdate
+      livenessProbe:
+        port: 8081
+      # Values for test - web
+      custom:
+    - name: "batch"
+      nodeType: "BackgroundProcessing,Search,Batch,RealTime,Custom1,Custom2,Custom3,Custom4,Custom5,BIX"
+      replicas: 1
+      deploymentStrategy:
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
+        type: RollingUpdate
+      livenessProbe:
+        port: 8081
+      # Values for test - batch
+      custom:
+    - name: "stream"
+      nodeType: "Stream"
+      requestor:
+        passivationTimeSec: 900
+      replicas: 1
+      volumeClaimTemplate:
+        resources:
+          requests:
+            storage: 5Gi
+      livenessProbe:
+        port: 8081
+      # Values for test - stream
+      custom:
```

Comment:
why do we even end on this empty `custom:` if it doesn't have an associated value? In added test, and pega-helm-charts/terratest/src/test/pega/pega-tier-deployment_test.go I don't actually see any requirement/verification that global.tier.custom is set

Reply:
FWIW, looks like as of latest commit it's past the yamllint issues. So if all goes well with the build, no need to change this, mainly just curious why we have a `custom` for any of these deployment spec, since it's never checked or enforced as far as I can tell

Reply:
There is no actual need for it - it can be removed. I think the origin was this test yaml being copied from the custom sidecar yaml, which included this custom block.